homework_22/generator.py

```diff
@@ -0,0 +1,30 @@
+import random
+
+from english_words import get_english_words_set
```

Comment:
ModuleNotFoundError: No module named 'english_words'

requirements.txt file is a joke for you))?